settings/cfg_Mission.sqf

```diff
@@ -50,5 +50,5 @@ var_keepRole = true; // Should players keep their role upon respawn? (default: t
 
 // MARKERS ////////////////////////////////////////////////////////////////////////////////////////
 var_markerSide = "ColorWEST"; // What colorsheme should markers on playerside use? ("ColorWEST", "ColorEAST", "ColorGUER") (default: "ColorWEST")
-var_groupMarkers = true; // Use group-icons on map. (default: true)
+var_groupTracker = true; // Use group-icons on map. (default: true)
```

Comment:
I hope no one is copy pasting their `cfg_Mission.sqf` from another mission then ;)

Reply:
> 
> 
> I hope no one is copy pasting their `cfg_Mission.sqf` from another mission then ;)

I don't think that will be an issue at this point (hopefully)